classes/blocks/class-guestbook.php

```diff
@@ -29,6 +29,9 @@ public function __construct() {
 		register_block_type(
 			self::get_full_block_name(),
 		);
+
+		add_action( 'enqueue_block_editor_assets', [ self::class, 'enqueue_editor_assets' ] );
```

Comment:
What if you directly invoke `self::enqueue_editor_assets` instead of passing the class and method name?

Reply:
Then you get an error, something like `"enqueue_editor_assets" is not a constant`, seems something like `function () { self::enqueue_editor_assets(); }` would work too but then it's longer so I think the current notation is better?